brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
+pref("browser.contentblocking.cryptomining.preferences.ui.enabled", false);
+pref("browser.contentblocking.fingerprinting.preferences.ui.enabled", false);
 
-/* 0320: disable about:addons' Recommendations pane (uses Google Analytics) ***/
-pref("extensions.getAddons.showPane", false); // [HIDDEN PREF]
-// Recommendations in about:addons' addons pane.
-pref("extensions.htmlaboutaddons.recommendations.enabled", false);
+// Remove addons.mozilla.org from set of domains that extensions cannot access
+pref("extensions.webextensions.restrictedDomains", "accounts-static.cdn.mozilla.net,accounts.firefox.com,addons.cdn.mozilla.net,api.accounts.firefox.com,content.cdn.mozilla.net,discovery.addons.mozilla.org,install.mozilla.org,oauth.accounts.firefox.com,profile.accounts.firefox.com,support.mozilla.org,sync.services.mozilla.com");
 
-/* 0330: disable telemetry
- * the pref (.unified) affects the behaviour of the pref (.enabled)
- * IF unified=false then .enabled controls the telemetry module
- * IF unified=true then .enabled ONLY controls whether to record extended data
- * so make sure to have both set as false
- * [NOTE] FF58+ 'toolkit.telemetry.enabled' is now LOCKED to reflect prerelease
- * or release builds (true and false respectively), see [2]
- * [1] https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/internals/preferences.html
- * [2] https://medium.com/georg-fritzsche/data-preference-changes-in-firefox-58-2d5df9c428b5 ***/
+/** TELEMETRY, STUDIES, EXPERIMENTS, CRASH REPORTS ***/
+// See https://github.com/yokoffing/Better-Fox/blob/a458e14a94ff67111868178639c188a5d08f7038/SecureFox.js#L510
+// for details.
+// [NOTE] We may need re-enable and/or redirect some of these for Ghostery Dawn.
+pref("datareporting.policy.dataSubmissionEnabled", false);
+pref("datareporting.healthreport.uploadEnabled", false);
+pref("browser.ping-centre.telemetry", false);
+pref("default-browser-agent.enabled", false);
+pref("app.shield.optoutstudies.enabled", false);
+pref("app.normandy.enabled", false);
+pref("app.normandy.api_url", "");
 pref("toolkit.telemetry.unified", false);
-pref("toolkit.telemetry.enabled", false); // see [NOTE] above FF58+
-pref("toolkit.telemetry.server", "data:,");
+pref("toolkit.telemetry.enabled", false);
 pref("toolkit.telemetry.archive.enabled", false);
-pref("toolkit.telemetry.newProfilePing.enabled", false); // [FF55+]
-pref("toolkit.telemetry.shutdownPingSender.enabled", false); // [FF55+]
-pref("toolkit.telemetry.updatePing.enabled", false); // [FF56+]
-pref("toolkit.telemetry.bhrPing.enabled", false); // [FF57+] Background Hang Reporter
-pref("toolkit.telemetry.firstShutdownPing.enabled", false); // [FF57+]
- /* 0331: disable Telemetry Coverage
-  * [1] https://blog.mozilla.org/data/2018/08/20/effectively-measuring-search-in-firefox/ ***/
-pref("toolkit.telemetry.coverage.opt-out", true); // [HIDDEN PREF]
-pref("toolkit.coverage.opt-out", true); // [FF64+] [HIDDEN PREF]
```

Comment:
please bring it back

Reply:
`pref("toolkit.telemetry.coverage.opt-out", true);` is also already in `firefox-brand.js` [here](https://github.com/ghostery/user-agent-desktop/blob/0e258946aa79eaadbb6fbc0d40899e64c0eb638a/brands/ghostery/branding/pref/firefox-branding.js#L85).

Restored `pref("toolkit.coverage.opt-out", true);` back to the file.

I'll double-check the telemetry settings again when I add comments to all the prefs.